src/prpy/planning/vectorfield.py

```diff
@@ -526,11 +526,13 @@ def fn_callback(t, q):
                 if path.GetNumWaypoints() == 1:
                     checks = [(t, q)]
                 else:
-                    # TODO: This should start at t_check. Unfortunately, a bug
-                    # in GetCollisionCheckPts causes this to enter an infinite
-                    # loop.
-                    checks = GetCollisionCheckPts(robot, path,
-                                                  include_start=False)
+                    # TODO: This will recheck the entire trajectory
+                    #  Ideally should just check the new portion of the trajectory
+                    from prpy.util import VanDerCorputSampleGenerator
+                    vdc = VanDerCorputSampleGenerator
+                    checks = GetLinearCollisionCheckPts(robot, path,
+                                                        norm_order=2,
+                                                        sampling_func=vdc)
```

Comment:
Can you modify `FollowVectorField` to accept a `sampling_func`? The default can still be `VanDerCorputSampleGenerator`. This is the convention we've followed in other planners to allow the user to configure this option.